### PR TITLE
Support any iterable object in keras.fit_generator

### PR DESCRIPTION
**Pull request by noamwies**

### CODE REVIEW

**Comment by noamwies** [comment]
I signed it!

**Comment by omalleyt12** [comment]
Thanks for the PR! This code path is actually being deprecated in 2.0 in favor of keras/engine/training_v2.py 

If you want to add support for this, it should be there

**Comment by gbaned** [comment]
@noamwies Could you please address the reviewer comments. Thanks!

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
